sentences/de/_common.yaml

```diff
@@ -288,6 +310,20 @@ expansion_rules:
   sind_wurden: "(sind|wurden)"
   position: "{position} [Prozent|%]"
   volume: "{volume_level} [Prozent|%]"
+
+  # Timers
+  timer_set: "(starte|setze|stelle|erstelle)"
+  timer_cancel: "(beende|stoppe)"
+  timer_duration_seconds: "{timer_seconds:seconds} Sekunde[n]"
+  timer_duration_minutes: "{timer_minutes:minutes} Minute[n][ [und ]{timer_seconds:seconds} Sekunde[n]]"
+  timer_duration_hours: "{timer_hours:hours} Stunde[n][ [und ]{timer_minutes:minutes} Minute[n]][ [und ]{timer_seconds:seconds} Sekunde[n]]"
```

Comment:
```suggestion
  timer_duration_minutes: "{timer_minutes:minutes} Minute[n] [[und]{timer_seconds:seconds} Sekunde[n]]"
  timer_duration_hours: "{timer_hours:hours} Stunde[n] [[und]{timer_minutes:minutes} Minute[n]] [[und]{timer_seconds:seconds} Sekunde[n]]"
```

Reply:
you forgot the space in [und ]. `12 Minuten und 5 Sekunden` would not match but `12 Minuten und5 Sekunden` would. I'll modify to keep tests successful.

Reply:
not in this case :grin: hassil does (_somehow_) add these spaces between optional words/expressions and expansion rules (_please, don't ask me about the exact mechanism in hassil during parsing the sentences_ :see_no_evil: ) ... further these suggestions are tested :wink:

Reply:
interesting. sorry, wasn't aware these were tested